src/sgx/libpoet_bridge/base64.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+base64.cpp and base64.h
```

Comment:
Please keep a list of external files used in our code. These will be required to be provided to Harsha when he updates the IP plan for the 1.2 release.

Reply:
Yeah, will keep list of external files.